src/config/src/utils/schema.rs

```diff
@@ -269,16 +269,14 @@ pub fn format_partition_key(input: &str) -> String {
 
 // format stream name
 pub fn format_stream_name(stream_name: &str) -> String {
+    let mut stream_name = RE_CORRECT_STREAM_NAME
+        .replace_all(stream_name, "_")
+        .to_string();
     if crate::get_config().common.format_stream_name_to_lower {
-        RE_CORRECT_STREAM_NAME
-            .replace_all(stream_name, "_")
-            .to_string()
-            .to_lowercase()
-    } else {
-        RE_CORRECT_STREAM_NAME
-            .replace_all(stream_name, "_")
-            .to_string()
+        stream_name.make_ascii_lowercase();
```

Comment:
https://doc.rust-lang.org/std/ascii/trait.AsciiExt.html#tymethod.make_ascii_lowercase
This method is depricated.

Reply:
Actually it's not.

The one you're referring to comes from `trait AsciiExt` and was deprecated because the [same method is directly available on `str` type](https://doc.rust-lang.org/std/primitive.str.html#method.make_ascii_lowercase), which is the one we're using.